app/lib/actions/katello/repository/sync.rb

```diff
@@ -26,7 +26,6 @@ def plan(repo, pulp_sync_task_id = nil, options = {})
           pulp_sync_options = {}
           pulp_sync_options[:download_policy] = ::Runcible::Models::YumImporter::DOWNLOAD_ON_DEMAND if validate_contents
           pulp_sync_options[:force_full] = true if skip_metadata_check
-          pulp_sync_options[:auto_publish] = false if skip_metadata_check #skip auto publish, and force it later
```

Comment:
nevermind, proceed with this.  I chatted with the pulp team and this option won't do anything